src/gui/widgets/sio1.cc

```diff
@@ -0,0 +1,390 @@
+    {"Stop bit length", "(0=Reserved/1bit, 1=1bit, 2=1.5bits, 3=2bits)"},  //
+    {"Not used", "(always zero)"},                                         //
+    {"Not used", "(always zero)"},                                         //
+    {"Not used", "(always zero)"},                                         //
+    {"Not used", "(always zero)"},                                         //
+    {"Not used", "(always zero)"},                                         //
+    {"Not used", "(always zero)"},                                         //
+    {"Not used", "(always zero)"},                                         //
+    {"Not used", "(always zero)"}                                          //
+};
+
+static registerDisplay controlDisplay[kControlEntries] = {
+    {"TX Enable (TXEN)", "(0=Disable, 1=Enable, when CTS=On)"},                      //
+    {"DTR Output Level", "(0=Off, 1=On)"},                                           //
+    {"RX Enable (RXEN)", "(0=Disable, 1=Enable)  ;Disable also clears RXFIFO"},      //
+    {"TX Output Level", "(0=Normal, 1=Inverted, during Inactivity & Stop bits)"},    //
+    {"Acknowledge", "(0=No change, 1=Reset SIO_STAT.Bits 3,4,5,9) (W)"},             //
+    {"RTS Output Level", "(0=Off, 1=On)"},                                           //
+    {"Reset", "(0=No change, 1=Reset most SIO_registers to zero) (W)"},              //
+    {"Unknown?", "(read/write-able when FACTOR non-zero) (otherwise always zero)"},  //
+    {"RX Interrupt Mode", "(0..3 = IRQ when RX FIFO contains 1,2,4,8 bytes)"},       //
+    {"RX Interrupt Mode", "(0..3 = IRQ when RX FIFO contains 1,2,4,8 bytes)"},       //
+    {"TX Interrupt Enable", "(0=Disable, 1=Enable) ;when SIO_STAT.0-or-2 ;Ready"},   //
+    {"RX Interrupt Enable", "(0=Disable, 1=Enable) ;when N bytes in RX FIFO"},       //
+    {"DSR Interrupt Enable", "(0=Disable, 1=Enable) ;when SIO_STAT.7  ;DSR=On"},     //
+    {"Not used", "(always zero)"},                                                   //
+    {"Not used", "(always zero)"},                                                   //
+    {"Not used", "(always zero)"}                                                    //
+};
+
+static ImGuiTableFlags tableFlags = ImGuiTableFlags_SizingFixedFit | ImGuiTableFlags_RowBg | ImGuiTableFlags_Borders |
+                                    ImGuiTableFlags_Resizable | ImGuiTableFlags_Reorderable | ImGuiTableFlags_Hideable;
+
+void ShowHelpMarker(const char* desc) {
+    ImGui::SameLine();
+    ImGui::TextDisabled("(?)");
+    if (ImGui::IsItemHovered()) {
+        ImGui::BeginTooltip();
+        ImGui::PushTextWrapPos(ImGui::GetFontSize() * 35.0f);
+        ImGui::TextUnformatted(desc);
+        ImGui::PopTextWrapPos();
+        ImGui::EndTooltip();
+    }
+}
+
+void PCSX::Widgets::SIO1::DrawControlEditor(PCSX::sio1Registers* regs) {
+    bool register_set;
+    std::string label;
+
+    ImGui::Text("Control: 0x%04x", regs->control);
```

Comment:
For translation support:
```suggestion
    ImGui::Text(_("Control: 0x%04x"), regs->control);
```